A32NX/de-DE.locPak

```diff
@@ -9,6 +9,12 @@
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_LEFT": "Ausgewählten Steuerkurs nach links anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
       "COCKPIT.TOOLTIPS.AUTOPILOT_PANEL_HDG_TUNING_RIGHT": "Ausgewählten Steuerkurs nach rechts anpassen (%((A:AUTOPILOT HEADING LOCK DIR:1,degrees))%!d!)",
       "COCKPIT.TOOLTIPS.APU_AVAILABLE": "APU verfügbar"
+      "A32NX.LOADING.TIPS.A320NEO_001": "Der FlyByWire A32NX ist das Ergebnis der Zusammenarbeit von Hunderten von individuellen Mitarbeitern."
+      "A32NX.LOADING.TIPS.A320NEO_002": "Wussten Sie schon? Die A320 kann bei Seitenwinden bis zu 38 Knoten und Rückenwinden bis zu 15 Knoten landen!"
```

Comment:
Please use "Der A320" instead of "Die" as you used the male form in all the other sentences and an Airbus is usually a male plane in german :P

Reply:
Done! Thank's for pointing this out.